channels/tsmf/client/tsmf_codec.c

```diff
@@ -255,6 +255,119 @@ static const TSMFMediaTypeMap tsmf_sub_type_map[] =
 
 };
 
+
+#define WaveFormatTag(_arg_) _arg_
```

Comment:
@hardening I would think you could easily grep for it instead of just searching for the numeric value.